test/utilities/connectivity_testing.go

```diff
@@ -199,19 +296,25 @@ func CreateTestNetwork(name string, netType string, ipams []hcn.Ipam, tryGetNetA
 	return network
 }
 
-func MakeTestStruct(t *testing.T, testNetwork *hcn.HostComputeNetwork, pluginType string, epPols bool, needGW bool, cid string) *PluginUnitTest {
+func MakeTestStruct(t *testing.T, testNetwork *hcn.HostComputeNetwork,
```

Comment:
same feedback as above about long function signatures. whatever you end up going with there, it would be good to apply that here as well